src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -202,21 +202,56 @@ let previous_map : (int * (int * (string * string))) list ref = ref []
 let phypath_to_sr_vdi : ((string, (string * string)) Hashtbl.t) = Hashtbl.create 20
 
 let refresh_phypath_to_sr_vdi () =
+  let iter_sr_dirs path f =
+    try
+      Utils.list_directory_entries_unsafe path |> List.iter (fun sruuid ->
```

Comment:
I'm not sure why we have `Utils.list_directory_entries_unsafe` when `Sys.readdir` seems good enough.